src/sentry/event_manager.py

```diff
@@ -810,9 +808,15 @@ def save(self, project_id, raw=False, assume_normalized=False):
             skip_consume=raw,
         )
 
-        metrics.timing("events.latency", received_timestamp - recorded_timestamp)
+        metric_tags = {"from_relay": "_relay_processed" in self._data}
 
-        metrics.timing("events.size.data.post_save", event.size)
+        metrics.timing("events.latency", received_timestamp - recorded_timestamp, tags=metric_tags)
+        metrics.timing("events.size.data.post_save", event.size, tags=metric_tags)
+        metrics.timing(
```

Comment:
This should be an incr

Reply:
Then we can only see errors/time-unit while this can give us avg/min/max value

Reply:
the one i want is "does sum of relay/non relay path" diverge. So i think we need the incr.

Reply:
nvm, you're right. max/min/avg of incr is only useless when you incr by 1 everytime